src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -317,7 +316,7 @@ public static function create_fragment( $html, $context = '<body>', $encoding =
 
 		$context_node = new WP_HTML_Token(
 			'context-node',
-			$processor->state->context_node[0],
+			'BODY',
```

Comment:
something seems off here: why are we hard-coding `BODY` as the context node? it seems like we missed this in line 301 when we added support for other contexts, whereas that line should have been updated to something like `$...->context_node[0]` but now we’re moving in the opposite direction…?

Reply:
This landed before the additional context support and I believe all these lines are now stale.

In this change, the `<body>` context was still hard-coded above. It moved out of the `->state->context_node` and into this location, while remaining hard-coded.

Reply:
The relevant lines are now around here:

https://github.com/WordPress/wordpress-develop/blob/4b3436968d60fc4dc301f3fe1ac888fbe6d59bce/src/wp-includes/html-api/class-wp-html-processor.php#L488
https://github.com/WordPress/wordpress-develop/blob/4b3436968d60fc4dc301f3fe1ac888fbe6d59bce/src/wp-includes/html-api/class-wp-html-processor.php#L538-L540